annotated_books.py

```diff
@@ -251,6 +249,9 @@ def __init__(self, parent, book_list, get_annotations_as_HTML, source):
                 if confidence is None:
                     cid, confidence = parent.generate_confidence(book_data)
 
+            enabled = QCheckBox()
+            enabled.setChecked(confidence)
+
```

Comment:
Several problems here:

1. unrelated change (or even feature) > move to another PR with an explanation and benefits
2. the `confidence` variable has a numerical value (from `0` to `5`), but the `enabled.setChecked(...)` method accepts a boolean value, which might lead to a type error > suggesting to replace `confidence` with `confidence > 0` (hopefully that will yield a boolean value as a result)